grades/README.md

```diff
@@ -127,5 +127,36 @@ and type (being ABSOLUTELY sure of what you are doing):
     FinalGrade.objects.filter(course_run__edx_course_key=edx_course_key).delete()
     CourseRunGradingStatus.objects.filter(course_run__edx_course_key=edx_course_key).delete()
 
+## How to adjust (curve) proctored exam grades
 
+Overview: After the last day of proctored exams, we deliver the grades to the course team. It typically takes them a few 
+weeks to adjust the grades and return them to us for upload. 
 
+### Delivering a preliminary grade report to the course team
+
+Grades arrive daily from the proctored exam vendor. Our Business Intelligence Server (BI) has a report that is updated 
+daily with grades. When the exam period is complete, we notify the course team so they can download a copy of the 
+proctored exam grades for analysis when the exam period is complete. 
+
+### Testing adjusted grades
+
+The course team will deliver updated grades to us via a csv file in dropbox. Since adjusting grades is a complex process,
```

Comment:
If it is not overkill, should we use keybase instead of dropbox for the csv file?

Reply:
Pros/Cons of dropbox/keybase:

Dropbox: 

Pros: 
  - supported by IS&T
  - simpler set up for new users
  - if we use MIT dropbox, then staff lose access when they leave MIT
  - access is auditable (I think?)

Cons: 
  - file is copied, unencrypted to local disk

Keybase:

Pros:
  - Data encrypted at rest (I think? Can we check?)

Cons:
  - not officially supported
  - expired accounts need to be removed manually (by revoking trust)
  - is there auditing?

@mitodl/devops do you guys have an opinion on this?

Reply:
This is very comprehensive comparison for Dropbox and keybase. We should copy it to a more accessible place so we can use it as a reference for the future.